apps/condo/domains/news/components/NewsPreview.tsx

```diff
@@ -0,0 +1,190 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react'
+import styled from '@emotion/styled'
+import { Row, Col, Image, Divider } from 'antd'
+import React, { useCallback, useMemo, useState } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Typography, Card, Space, RadioGroup, Radio } from '@open-condo/ui'
+import { colors } from '@open-condo/ui/dist/colors'
+
+import { CrossIcon } from '@condo/domains/common/components/icons/CrossIcon'
+import { DEFAULT_BORDER_RADIUS } from '@condo/domains/common/constants/style'
+
+import type { NewsItem } from '@app/condo/schema'
+import type { RowProps } from 'antd'
+
+enum NewsPreviewTabTypes {
+    Push = 'push-notification',
+    App = 'resident-app',
+}
+type NewsPreviewProps = Pick<NewsItem, 'title' | 'body' | 'validBefore'>
+
+interface INewsPreview {
+    (props: NewsPreviewProps): React.ReactElement
+}
+
+interface INewsPushPreview {
+    (props: Omit<NewsPreviewProps, 'validBefore'>): React.ReactElement
+}
+
+const PUSH_ROW_GUTTER: RowProps['gutter'] = [0, 8]
+const PUSH_ROW_CONTENT_GUTTER: RowProps['gutter'] = [0, 4]
+const PUSH_DATETIME_TEXT_STYLE: React.CSSProperties = { textAlign: 'right' }
+const APP_TOP_COLUMN_STYLE: React.CSSProperties = { textAlign: 'center', paddingTop: '12px' }
+const APP_CLOSE_ICON_STYLE: React.CSSProperties = { position: 'absolute', right: '10px', top: '14px' }
+const APP_CONTENT_STYLE: React.CSSProperties = { padding: '0 12px' }
+const PUSH_PARAGRAPH_ELLIPSIS_CONFIG = { rows: 2 }
+const PREVIEW_CONTENT_WIDTH = 360
+
+const NewsPreviewContainer = styled.div`
```

Comment:
Some tech. talks ago we've decided to avoid emotion, why is it here?

Reply:
We have an issue of usage built-in css-modules support, because we already use custom style loader plugins such as `@zeit/next-css` and `@zeit/next-less`. So first step to replace emotion with something else is migration from custom css loades.
I suppose it is not a part of this task

Reply:
More related information you can find at the blog of next-js https://nextjs.org/blog/next-9-2

Reply:
Added a TODO for the future investigation of that problem